src/App.vue

```diff
@@ -1,8 +1,11 @@
 <script setup lang="ts">
 import { useRouter, RouterLink, RouterView } from "vue-router";
 import { useMapStore } from "./stores/map";
+import { useUIStore } from "./stores/ui";
+import ContributionBanner from "./components/ContributionBanner.vue";
 
 const mapStore = useMapStore();
+const uiStore = useUIStore();
```

Comment:
This is a nicer solution for future/more complex UI stuff as well!

Reply:
Thank you!